pkgs/servers/http/apt-cacher-ng/default.nix

```diff
@@ -1,17 +1,30 @@
-{ stdenv, fetchurl, cmake, doxygen, zlib, openssl, bzip2, pkgconfig, libpthreadstubs }:
+{ stdenv
+, bzip2
+, cmake
+, doxygen
+, fetchurl
+, fuse
+, lzma
+, openssl
+, pkgconfig
+, systemd
+, tcp_wrappers
+, zlib
+}:
 
 stdenv.mkDerivation rec {
   name = "apt-cacher-ng-${version}";
-  version = "0.9.1";
+  version = "3.1";
 
   src = fetchurl {
     url = "http://ftp.debian.org/debian/pool/main/a/apt-cacher-ng/apt-cacher-ng_${version}.orig.tar.xz";
-    sha256 = "1d686knvig1niapc1ib2045f7jfad3m4jvz6gkwm276fqvm4p694";
+    sha256 = "0p8cdig70vz1dgw2v8brjin5wqrk8amncphyf11f53bza5grlc91";
   };
 
-  NIX_LDFLAGS = "-lpthread";
   nativeBuildInputs = [ cmake doxygen pkgconfig ];
-  buildInputs = [ zlib openssl bzip2 libpthreadstubs ];
+  buildInputs = [ bzip2 fuse lzma openssl systemd tcp_wrappers zlib ];
+
+  cmakeFlags = [ "-DCMAKE_BUILD_TYPE=Release" ];
```

Comment:
this is the default: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/build-managers/cmake/setup-hook.sh#L52

Reply:
fixed in latest version.